plugins/scaffolder/src/components/fields/RepoUrlPicker/RepoUrlPicker.tsx

```diff
@@ -201,6 +201,10 @@ export const RepoUrlPicker = (props: RepoUrlPickerProps) => {
           rawErrors={rawErrors}
           state={state}
           onChange={updateLocalState}
+          accessToken={
```

Comment:
Im thinking that we should probably refactor these pickers so that they grab their own access token rather than passing through the tokens, it's a little unfortunate I think and it's not neccessary to prop drill as they should be cached and it wont open a new popup.